resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/internal/ClientResponse.java

```diff
@@ -305,6 +306,8 @@ else if (bufferedEntity == null)
             {
 
             }
+            // record error details in server.log

```

Comment:
Shouldn't this be "client log"? @marekkopecky said that, in the tests, the test is running on the server, so the message is going to the server log, but that wouldn't normally be the case, right?